### DISCUSSION TITLE
Allows to adjust the alpha of each hud component(or hud overlay?)

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#22475] [Ideas]
Can provide information while reducing interference with reading beatmap

Why:
Under normal circumstances, most of the component are in the position where the mania note appears (related to "Use different skins for different rulesets"), But there's some possibility, provides an option to adjust its alpha, players can obtain information without affecting the reading of the picture

https://user-images.githubusercontent.com/34775378/215945478-d685b8fa-f4ac-43cb-a792-24c8ba21d47e.mp4

**Comment by peppy** [comment]
I think this should be fine to implement in some way. Of note though, some components will not look good with alpha applied (unless buffered) due to alpha being applied individually to each layer.
